index.html

```diff
@@ -166,6 +166,21 @@
 If certain groups are deliberately excluded from using these features, explain
 why.
 
+<aside class="example" title="Excluded Users" id="example-exclusion">
+
+Example: The [Web GPU specification](https://www.w3.org/TR/webgpu/)
+provides low-level access to developers, to allow them to render
+high-performance graphics like advanced 3D-rendering.
+The GPU for the Web Community Group, have identified that over
+(half a billion users are unable to access WebGPU rendered objects)[https://github.com/explainers-by-googlers/webgpu-compatibility-mode/blob/main/README.md#user-facing-problem] 
+because older devices lack support for modern graphics APIs.
+To address this, the Community Group have proposed
```

Comment:
Since it's in the spec now:

```suggestion
To address this, the Working Group defined
```